README.adoc

```diff
@@ -1,101 +1,178 @@
+ifdef::env-github[:check: :ballot_box_with_check:]
+ifndef::env-site[:status:]
+:uri-actdiag: http://blockdiag.com/en/actdiag/index.html
+:uri-asciidoctor-api: http://asciidoctor.org/docs/user-manual/#api
+:uri-asciidoctor-extensions: http://asciidoctor.org/docs/user-manual/#extension-points
+:uri-blockdiag: http://blockdiag.com
+:uri-ditaa: http://ditaa.sourceforge.net/
+:uri-dot: http://www.graphviz.org/content/dot-language
+:uri-graphviz: http://www.graphviz.org
+:uri-imagemagick: http://www.imagemagick.org
+:uri-java: http://java.sun.com
+:uri-mermaid: http://knsv.github.io/mermaid/
+:uri-nwdiag: http://blockdiag.com/en/nwdiag/index.html
+:uri-packetdiag: http://blockdiag.com/en/nwdiag/index.html
+:uri-phantomjs: http://phantomjs.org
+:uri-plantuml: http://plantuml.sourceforge.net
+:uri-py-plantuml: https://code.google.com/p/asciidoc-plantuml/
+:uri-rackdiag: http://blockdiag.com/en/nwdiag/index.html
+:uri-seqdiag: http://blockdiag.com/en/seqdiag/index.html
+:uri-shaape: https://github.com/christiangoltz/shaape
+:uri-wavedrom: http://wavedrom.com
+:uri-wavedromeditor: https://github.com/wavedrom/wavedrom.github.io/releases
+:uri-wavedromcli: https://github.com/wavedrom/cli
+
+Asciidoctor Diagram is a set of extensions for Asciidoctor that enable you to add diagrams to your AsciiDoc document which you describe using plain text.
+
+The extensions support BlockDiag (BlockDiag, SeqDiag, ActDiag, NwDiag), Ditaa, GraphViz, Mermaid, PlantUML, Shaape and WaveDrom syntax.
+
+Each extension runs the diagram processor to generate an SVG, PNG, or TXT file from the input text.
+The generated file is then inserted into your converted document.
+
+Asciidoctor Diagram was inspired by the {uri-py-plantuml}[AsciiDoc PlantUML filter].
+
+ifdef::status[]
 == Status
 
 image:https://travis-ci.org/asciidoctor/asciidoctor-diagram.svg?branch=master["Linux Build Status", link="https://travis-ci.org/asciidoctor/asciidoctor-diagram"]
 
 image:https://ci.appveyor.com/api/projects/status/4r4gkk5gy3igs6nh/branch/master?svg=true["Windows Build Status", link="https://ci.appveyor.com/project/asciidoctor/asciidoctor-diagram"]
+endif::status[]
 
-== Installation
+ifeval::["{toc-placement}" == "macro"]
+[discrete]
+== Contents
 
-Add this line to your application's Gemfile:
+toc::[title={blank}]
+endif::[]
 
-```ruby
-gem 'asciidoctor-diagram'
-```
+== Installation
 
-And then execute:
+Asciidoctor Diagram is a RubyGem, which can be installed using the `gem` or `bundle` commands.
 
- $ bundle
-
-Or install it yourself as:
+You can install the Asciidoctor Diagram gem by typing `gem install` in the CLI.
 
  $ gem install asciidoctor-diagram
 
-=== Additional Requirements
+by first adding the following entry to your project's [.path]_Gemfile_.
 
-Certain diagram types require other tools to be installed seperately.
+.Gemfile
+[source,ruby]
+----
+gem 'asciidoctor-diagram', '~> 1.4.0' // <1>
+----
+<1> Add this line to your application's Gemfile and save the file.
```

Comment:
I don't think this callout is needed. It says the same thing as the sentence before it and the listing only has one line, so it's hard to confuse.